### DISCUSSION TITLE
Is it possible to geta link?

### DISCUSSION THREAD

**Discussion by stefs22** [streamlink/streamlink#5181] [Q&A] [answered]
### Description

It worked on, i just put streamlink "STREAMURL" 1080p and it worked. But, if is possible to get a link so i can use this on every software i want and how?

**Comment by bastimeyer** [comment, marked as answer]
https://streamlink.github.io/cli.html#cmdoption-stream-url

**Comment by stefs22** [comment]
Are link is permanent or will stop working after some hours?
